From: Bart Van Assche <bart.vanassche@sandisk.com>
Date: Tue, 15 Nov 2016 11:21:46 -0800
Subject: qla2xxx: Add an #include directive
References: bsc#1019675,FATE#321701
Git-commit: 09ce66aee7763686a22ab26fbe8a889e0cec64e9
Patch-Mainline: v4.10-rc1

The following statement in qla_isr.c needs the size of struct
t10_pi_tuple:

    spt += j;

Hence include the <linux/t10-pi.h> header file.

Signed-off-by: Bart Van Assche <bart.vanassche@sandisk.com>
Acked-by: Himanshu Madhani <himanshu.madhani@cavium.com>
Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 drivers/scsi/qla2xxx/qla_isr.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/scsi/qla2xxx/qla_isr.c b/drivers/scsi/qla2xxx/qla_isr.c
index a812606..fe73ba7 100644
--- a/drivers/scsi/qla2xxx/qla_isr.c
+++ b/drivers/scsi/qla2xxx/qla_isr.c
@@ -9,6 +9,7 @@
 
 #include <linux/delay.h>
 #include <linux/slab.h>
+#include <linux/t10-pi.h>
 #include <scsi/scsi_tcq.h>
 #include <scsi/scsi_bsg_fc.h>
 #include <scsi/scsi_eh.h>
-- 
1.8.5.6

